system-variables.md

```diff
@@ -142,12 +166,38 @@ mysql> SELECT * FROM t1;
 - This variable sets the authentication method that the server advertises when the server-client connection is being established. Possible values for this variable are documented in [Authentication plugin status](/security-compatibility-with-mysql.md#authentication-plugin-status).
 - Value options: `mysql_native_password` and `caching_sha2_password`. For more details, see [Authentication plugin status](/security-compatibility-with-mysql.md#authentication-plugin-status).
 
+### default_week_format
+
+- Scope: SESSION | GLOBAL
+- Default value: `0`
+- Range: `[0, 7]`
+- Set the week format used by the `WEEK()` function
+
 ### foreign_key_checks
 
 - Scope: SESSION | GLOBAL
 - Default value: `OFF`
 - For compatibility, TiDB returns foreign key checks as `OFF`.
 
+### group_concat_max_len
+
+- Scope: SESSION | GLOBAL
+- Default value: `1024`
+- Range: `[4, 18446744073709551615]`
+- The maximum buffer size for items in the `GROUP_CONCAT()` function.
+
+### have_openssl
+
+- Scope: NONE
+- Default value: `DISABLED`
+- MySQL compatibiltiy read-only variable. Set to `YES` when the server has TLS enabled.
+
+### have_ssl
+
+- Scope: NONE
+- Default value: `DISABLED`
+- MySQL compatibiltiy read-only variable. Set to `YES` when the server has TLS enabled.
```

Comment:
```suggestion
- A read-only variable for MySQL compatibility. You can set it to `YES` when the server has TLS enabled.
```

Reply:
This is set by the server, not by the user.